index.bs

```diff
@@ -413,9 +413,14 @@ The user agent may delay the reporting of layout shifts after a
 until such time as it is known that the event does not begin a fling or scroll
 gesture.
 
-The <a href="https://www.w3.org/TR/uievents/#event-type-mousemove">mousemove</a> and
-<a href="https://www.w3.org/TR/pointerevents/#the-pointermove-event">pointermove</a>
-events are also not excluding inputs.
+The <a href="https://www.w3.org/TR/uievents/#event-type-mousemove">mousemove</a> events after the
```

Comment:
Nit: I think what we want is: pointermove after pointerdown that is not followed by pointercancel, right? I.e. a way the current wording could be interpreted is that the following sequence would consist of an excluding pointermove:
```pointerdown, pointermove, pointercancel...```
And I think we want that pointermove to not count as an excluding input, right? So in a sense we kinda need to wait until we know that pointercancel won't be fired in order to know that pointermove was excluding. Does this match the implementation, and does it make sense to you?

Reply:
I have made some changes, now pointermove events are not excluding inputs.